src/System.Net.Primitives/tests/PalTests/System.Net.Primitives.Linux.Pal.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <TargetsWindows>false</TargetsWindows>
+    <TargetsLinux>true</TargetsLinux>
+    <TargetsOSX>false</TargetsOSX>
+    <TargetsFreeBSD>false</TargetsFreeBSD>
+    <TargetsUnix>true</TargetsUnix>
+    <UnsupportedPlatforms>Windows_NT;OSX;FreeBSD</UnsupportedPlatforms>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)\System.Net.Primitives.Pal.Tests.projinc" />
```

Comment:
@pgavlin I have a few observations regarding the Import:
1. We were using the .msbuild extension - I'd like to be consistent and use the same msbuild include extension throughout our sources. I have no preferences on the name.
2. There's a bug in VS that would break showing files correctly in Solution Explorer unless the files are referenced directly in the root project file. There's also a workaround of creating a new tag and adding the files through that one. See `System.Net.Http.csproj` and the `ItemGroup` that contains `<Compile Include="@(CompileItem)" />`